source/btle/btle.cpp

```diff
@@ -183,6 +183,7 @@ static void btle_handler(ble_evt_t *p_ble_evt)
                     break;
             }
             nRF5xGap::getInstance().processDisconnectionEvent(handle, reason);
+            // TODO: close pending discoveries
```

Comment:
This is a question. Is it required to close pending discovery manually ? If a discovery is running, does the stack will report the discovery that the discovery has failled (service, characteristic and characteristic descriptors) ?